CMakeLists.txt

```diff
@@ -139,7 +140,7 @@ execute_process(
 if(DEFINED ENV{SDK_VERSION})
   set(OSQUERY_BUILD_SDK_VERSION "${ENV{SDK_VERSION}}")
 else()
-  string(REPLACE "-" ";" OSQUERY_BUILD_SDK_VERSION ${OSQUERY_BUILD_VERSION})
+  string(REPLACE "-" ";" OSQUERY_BUILD_SDK_VERSION "1.4.5")
```

Comment:
Is there any way for ports to set this variable? And then send it in to CMake within CMAKE_ARGS? Let me know what you think is an appropriate variable name to check. OSQUERY_EXTERNAL_VERSION, OSQUERY_MANUAL_VERSION? Something along those lines.

CMake can check for environment variables too. Either way, if it's set then CMake will skip trying to discover through git tags.